engine/src/main/java/org/hibernate/search/query/dsl/sort/MultiValuedMode.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ *
+ */
+package org.hibernate.search.query.dsl.sort;
+
+/**
+ * Mode when comparing multiple values.
+ *
+ * @author Emmanuel Bernard emmanuel@hibernate.org
+ */
+public enum MultiValuedMode {
```

Comment:
Hum, elastic search does not document their default if they even have one. I imagine they pick the first unless specified ?